src/main/kotlin/org/danilopianini/gradle/mavencentral/PublishOnCentralExtension.kt

```diff
@@ -87,24 +88,32 @@ open class PublishOnCentralExtension(val project: Project) {
         repoOwner.map { "https://github.com/$it/${project.name}" }
     )
 
+    /**
+     * The style of `javadoc` artifacts being published on Maven repositories.
+     */
+    val docStyle: Property<DocStyle> = project.objects.property<DocStyle>().convention(
+        DocStyle.HTML
```

Comment:
```suggestion
        DocStyle.JAVADOC
```
Am I correct?

Reply:
Oh yes. I was thinking to my default, not yours. Apologies. JAVADOC is the correct default

Reply:
I force pushed a new version of the commit where the convention is set to JAVADOC